api/src/main/java/javax/enterprise/concurrent/package-info.java

```diff
@@ -15,6 +15,6 @@
  */
 
 /**
- * Classes and interfaces for concurrent programming in Java EE&trade; applications. 
+ * Classes and interfaces that make up the Jakarta Concurrency Specification. 
```

Comment:
Sometimes "specification" is capitalized and sometimes not.
Shouldn't it be consistent?  I would think that not capitalizing
it would be better so that people aren't tempted to include it
in an acronym, e.g., "JCS".

Reply:
Done